charts/passbolt-operator/README.md

```diff
@@ -0,0 +1,96 @@
+# passbolt-operator
+
+![Version: 1.1.0](https://img.shields.io/badge/Version-1.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v1.1.0](https://img.shields.io/badge/AppVersion-v1.1.0-informational?style=flat-square)
+
+A Helm chart to deploy the Passbolt Operator. The Passbolt Operator is a Kubernetes Operator to manage Passbolt Secrets in Kubernetes (Passbolt --> Kubernetes secret).
+
+## Installation
+
+### Add Helm repository
+
+```bash
+helm repo add passbolt-operator https://urbanmedia.github.io/passbolt-operator-helm
+```
+
+### Install the operator
+
+First, create the namespace where the operator will be installed:
+
+```bash
+kubectl create namespace passbolt-operator-system
+```
+
+Then, create the secret containing the Passbolt API credentials:
+
+```bash
+kubectl create secret generic controller-passbolt-secret \
+  --from-file=gpg_key='/path/to/gpg.key' \
+  --from-literal=password='<my-user-password>' \
+  --from-literal=url='<my-passbolt-url>' \
+  --namespace system
+```
+
+Finally, install the operator:
+
+```bash
+helm install \
+  passbolt-operator passbolt-operator/passbolt-operator \
+  -n passbolt-operator-system
+```
+
+### Upgrading from previous major version
+
+Since the operator is installed along with the CRDs, you need to force the upgrade to ensure the CRDs are updated:
+
+```bash
+helm upgrade \
+  passbolt-operator passbolt-operator/passbolt-operator \
+  -n passbolt-operator-system \
+  --force
+```
```

Comment:
> I get the following error when trying to upgrade the `passbolt-operator` release:
> 
> > Error: UPGRADE FAILED: rendered manifests contain a resource that already exists. Unable to continue with update: CustomResourceDefinition "passboltsecrets.passbolt.tagesspiegel.de" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "passbolt-operator"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "passbolt-operator-system"
> 
> After deleting the CRD manually, it works.

Unfortunately, I cannot change the behavior. This guide is intended to help users update their deployments.